src/main/scala/interplay/Versions.scala

```diff
@@ -8,4 +8,5 @@ object ScalaVersions {
 object SbtVersions {
   val sbt10 = "1.2.8"
```

Comment:
I would also remove those two. Projects using newer versions of interplay should then migrate to sbt 1.5.

Reply:
Removed.